nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -732,16 +651,13 @@ int ddxProcessArgument(int argc, char *argv[], int i)
       free(nxagentKeyboard);
       nxagentKeyboard = NULL;
 
+      /* FIXME: why limit to 256? */
```

Comment:
There's probably no good reason for that. A cursory look through the code doesn't reveal any obvious length limitation for the keyboard type/strings. Maybe drop that completely?

Reply:
done